docs/en/query_language/create.md

```diff
@@ -93,22 +93,23 @@ Besides default data compression, defined in [server settings](../operations/ser
 
 Supported compression algorithms:
 
-- `NONE` - no compression for data applied
+- `NONE` — no compression for data applied.
 - `LZ4`
-- `LZ4HC(level)` - (level) - LZ4\_HC compression algorithm with defined level.
+- `LZ4HC(level)` — (level) - LZ4\_HC compression algorithm with defined level.
 Possible `level` range: \[3, 12\]. Default value: 9. Greater values stands for better compression and higher CPU usage. Recommended value range: [4,9].
```

Comment:
two ranges are written inconsistently: one has backslashes and space, the other do not